actions.yaml

```diff
@@ -0,0 +1,5 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+get-postgres-password:
```

Comment:
One minor change

```suggestion
get_postgres_password:
```

Reply:
This will cause an `bad action name` error on deploy (we need to follow the action name rule from https://github.com/juju/charm/blob/a407f299325d3fcfdd249a9c104cb87184d61064/actions.go#L20).